modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding plugins to the {cli-manager} with YAML files
+
+You can add a CLI plugin to the {cli-manager} using the Form View or YAML View.
+
+[NOTE]
+====
+If you use the form view, add the information for the CLI plugin into the required fields, which are *Name*, *Platforms*, *Files*, *Image*, *Platform*, *Short description*, and *Version*. Then click *Create* to add the CLI plugin.
+====
```

Comment:
@cbippley May be adding as below would make it more clear i believe, WDYT ? 
Name,  platforms (bin, from), image, ImagePullSecret, platform, CA (in case of disconnected), Short description, version & homepage.

Reply:
These are all required? They are not all marked as such in the web console.